src/structures/guild_preference.ts

```diff
@@ -228,7 +243,8 @@ export default class GuildPreference {
         const preset: { [x: string]: any } = (await dbContext.kmq("game_option_presets")
             .select(["option_name", "option_value"])
             .where("guild_id", "=", this.guildID)
-            .andWhere("preset_name", "=", presetName))
+            .andWhere("preset_name", "=", presetName)
+            .andWhere("option_name", "!=", "uuid"))
```

Comment:
Don't even need to do this check, the uuid field will just go away when we cast to GameOption

Reply:
We still need to do this check because we perform a JSON.parse on each row and the UUID is stored normally

Reply:
It'll get validated out in `validateGameOptions`